internal/controller/ols_app_redis_assets.go

```diff
@@ -56,10 +101,22 @@ func (r *OLSConfigReconciler) generateRedisDeployment(cr *olsv1alpha1.OLSConfig)
 									corev1.ResourceMemory: resource.MustParse("1Gi"),
 								},
 							},
+							Command: []string{"redis-server",
+								"--port", "0",
+								"--tls-port", "6379",
```

Comment:
port 6379 can be defined in `constants.go`, as it is shared between deployment, service and tests.